site/_docs/history.md

```diff
@@ -29,22 +29,107 @@ Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
 
 <!--
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.37.0">1.37.0</a> / XXXX-XX-XX
-{: #v1-37-0}
 
-This release comes x months after [1.36.0](#v1-36-0),
-contains contributions from x contributors, and resolves x issues.
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.38.0">1.38.0</a> / XXXX-XX-XX
+{: #v1-38-0}
 
+This release comes x months after [1.37.0](#v1-37-0),
+contains contributions from x contributors, and resolves x issues.
 Contributors to this release:
 x,
 y (release manager),
 z.
 
+#### Breaking Changes
+{: #breaking-1-38-0}
+
+Compatibility: This release is tested on Linux, macOS, Microsoft Windows;
+using JDK/OpenJDK versions 8 to 19;
+Guava versions 21.0 to 32.1.3-jre;
+other software versions as specified in gradle.properties.
+
+#### New features
+{: #new-features-1-38-0}
+
+#### Dependency version upgrade
+{: #dependency-1-38-0}
+
+#### Bug-fixes, API changes and minor enhancements
+{: #fixes-1-38-0}
+
+#### Build and test suite
+{: #build-1-38-0}
+
+#### Web site and documentation
+{: #site-1-38-0}
+-->
+
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.37.0">1.37.0</a> / 2024-23-04
+{: #v1-37-0}
+
+This release comes 5 months after [1.36.0](#v1-36-0),
+contains contributions from 46 contributors, and resolves 136 issues. It's worth highlighting the
+introduction of adapter for Apache Arrow ([<a href="https://issues.apache.org/jira/browse/CALCITE-2040">CALCITE-2040</a>]),
+StarRocks dialect ([<a href="https://issues.apache.org/jira/browse/CALCITE-6257">CALCITE-6257</a>]).
+The release also added support for lambda expressions in SQL ([<a href="https://issues.apache.org/jira/browse/CALCITE-3679">CALCITE-3679</a>]),
```

Comment:
The evolution to support for lambda expressions (CALCITE-3679) added new methods to RexBiVisitor and RexVisitor ([6f64865](https://github.com/apache/calcite/commit/6f64865eb8c71a65bde60a19de2de42775fae4ed)), so perhaps it would be worth adding that in the "breaking changes" section? Something along the lines
```
As a consequence of the support for lambda expressions ([<a href="https://issues.apache.org/jira/browse/CALCITE-3679">CALCITE-3679</a>]) new methods have been added to `RexVisitor`and `RexBiVisitor`; any class implementing one of them will have to implement the new methods.
```